### DISCUSSION TITLE
Move Achievement Badges to a separate tab

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [community/community#28735] [Other Feature Feedback, Questions, & Ideas]
Remove achievement badges from the profile and move them to a separate tab

**Comment by radar-blip** [comment]
You dont have to display them if you dont want to. Its ok 😄

**Comment by NitinIsFit** [comment]
@thenithinbalaji your username says `nitin balaji` hi i am `nitin ojhaji` 😂😂
You must be from rajasthan right? Same state as me.

**Reply by thenithinbalaji** [reply]
Oh great 😄
